### PR TITLE
Update enable-controlled-folders.md

### LINKED ISSUE #11836: Config steps correction
[
https://learn.microsoft.com/en-us/microsoft-365/security/defender-endpoint/enable-controlled-folders?view=o365-worldwide#microsoft-intune

Correction Step#3 >> 
Platform > win10,win11 and win server
Profile >> ASR rules

image

]

---
#### Document Details

⚠ *Do not edit this section. It is required for learn.microsoft.com ➟ GitHub issue linking.*

* ID: 25faafb3-297a-dbb1-651b-dcf18081e95b
* Version Independent ID: 61a3c7ca-8094-5219-7f7c-688124bab07c
* Content: [Enable controlled folder access](https://learn.microsoft.com/en-us/microsoft-365/security/defender-endpoint/enable-controlled-folders?view=o365-worldwide#microsoft-intune)
* Content Source: [microsoft-365/security/defender-endpoint/enable-controlled-folders.md](https://github.com/MicrosoftDocs/microsoft-365-docs/blob/public/microsoft-365/security/defender-endpoint/enable-controlled-folders.md)
* Service: **microsoft-365-security**
* Sub-service: **mde**
* GitHub Login: @Dansimp
* Microsoft Alias: **dansimp**

### PR DESCRIPTION
**Pull request by msbemba**
Made changes to Step 3

Fixes https://github.com/MicrosoftDocs/microsoft-365-docs/issues/11836

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

File: microsoft-365/security/defender-endpoint/enable-controlled-folders.md [hunk 1]
```diff
@@ -77,7 +77,7 @@ For more information about disabling local list merging, see [Prevent or allow u
 
 2. Go to **Attack Surface Reduction** \> **Policy**.
 
-3. Select **Platform**, choose **Windows 10 and later**, and select the profile **Attack Surface Reduction rules** \> **Create**.
+3. Select **Platform**, choose **Windows 10, Windows 11 and Windows Server**, and select the profile **Attack Surface Reduction rules** \> **Create**.
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/enable-controlled-folders.md]:
```suggestion
3. Select **Platform**, choose **Windows 10, Windows 11, and Windows Server**, and select the profile **Attack Surface Reduction rules** \> **Create**.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@Dansimp Please merge this PR. Thanks

**Pull request merged**